tools/lkl/lib/hijack/hijack.c

```diff
@@ -283,3 +283,20 @@ int epoll_ctl(int epollfd, int op, int fd, struct epoll_event *event)
 
 	return lkl_call(__lkl__NR_epoll_ctl, 4, epollfd, op, fd, event);
 }
+
+int eventfd(unsigned int count, int flags)
+{
+	return lkl_sys_eventfd2(count, flags);
```

Comment:
What's the purpose of CHECK_HOST_CALL?  Isn't the host_xxx already initialized in `init2_host_##name`?